ui/resizable.js

```diff
@@ -173,13 +174,14 @@ $.widget("ui.resizable", $.ui.mouse, {
 
 				handle = $.trim(n[i]);
 				hname = "ui-resizable-" + handle;
-				axis = $("<div class='ui-resizable-handle " + hname + "'></div>");
+				axis = $("<div>");
+				this._addClass( axis, "ui-resizable-handle " + hname );
 
 				axis.css({ zIndex: o.zIndex });
 
 				// TODO : What's going on here?
 				if ("se" === handle) {
-					axis.addClass("ui-icon ui-icon-gripsmall-diagonal-se");
+					this._addClass( axis, null, "ui-icon ui-icon-gripsmall-diagonal-se" );
```

Comment:
As far the the to do whats doing on we ad an icon to the bottom corner but only if se is one of the handles. I don't think we should handle customization of icons through the classes option though since these are not theme classes. I think if we want to support customization here we should add an option consistent with other widget. However i also this this should wait for the pending re-writes @mikesherov  thoughts?